docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -110,3 +104,11 @@ To remove a standby instance from a high-availability cluster, you need to remov
 The standby instance is now a standalone instance again.
 
 After you have returned a standby instance to standalone mode, the information on the instance is likely to be out of date, which can lead to incorrect behavior. It is recommended to wipe out the state information before using it in standalone mode. For assistance with resetting the state of a standby instance that you removed from a high-availability cluster, contact Yugabyte Support.
+
+## Limitations
+
+If you are using custom ports for Prometheus in your YugabyteDB Anywhere installation and the YugabyteDB Anywhere instance is configured for high availability with other YugabyteDB Anywhere instances, then the following limitation applies: 
+
+- All YugabyteDB Anywhere instances configured under high availability must use the same custom port.
+
+  The default Prometheus port for YugabyteDB Anywhere is `9090`. Custom ports are configured through the settings section of the Replicated installer UI available at `https://<yugabyteanywhere-ip>:8800/`.
```

Comment:
1. Should we instead point to this page https://docs.yugabyte.com/preview/yugabyte-platform/install-yugabyte-platform/install-software/default/ for "How to access the replicated settings page to customise the port configuration"? This page currently does not show a screenshot of the custom port settings. However, a future edit to this page could consolidate a description of all install time settings that we have allowed on the Replicated UI.
2. Note that the Replicated installer UI is **normally** available at `https://<yugabyteanywhere-ip>:8800`. The port of the replicated UI itself can be configured to any other port, say 7700, during a replicated install using a command like this.
```
curl -sSL https://get.replicated.com/docker | sudo bash -s ui-bind-port=7700
```
In such a case the Replicated UI would have to be accessed at `https://<yugabyteanywhere-ip>:7700`.

Reply:
Added link and changed "UI is typically available"